src/build_scripts/build_lang.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -eu
+
+## preconditions
+if [ -z "${GERBIL_SETUP+x}" ]; then
+  (1>&2 echo "ERROR: Do not call this script directly.") && exit 1
```

Comment:
that's not correct, we want to be able to build individual components -- very useful in development.